porcupine/default_filetypes.toml

```diff
@@ -421,3 +421,11 @@ tabs2spaces = true
 indent_size = 4
 comment_prefix = '//'
 autoindent_regexes = {dedent = '(return( .+)?|break|continue);', indent = 'this regex matches nothing^'}
+[[Rust.example_commands]]
+command = "cargo run"
+command = "cargo test"
```

Comment:
Thanks! This looks promising!
However, you should put every command in its own `[[Rust.example_commands]]` table (you can see above, how is it done for Python commands for example), because this way there are duplicate `command` keys in a `[[Rust.example_commands]]`, and that isn't allowed.

Reply:
Oh thank you I will do the same.

Reply:
To test whether it works, you can:
1. Run porcupine
2. Make a file named `asdasd.rs`
3. Press Shift+F5

The command entry that appears should already have `cargo run` in it (taken from this file). As you type into it, other commands should be available with auto-complete, e.g. typing `cargo t` should auto-complete to `cargo test`.

Reply:
Regarding running porcupine from where do I run it. Is it from `laucher/lauch.pyw` ? Because then it is giving errors like 
`ModuleNotFoundError: No module named 'porcupine'`

Reply:
Regarding running `porcupine`, how do I do that, is it from `launcher/launch.pyw` because then it is showing `ModuleNotFoundError` like 
```
ModuleNotFoundError: No module named 'porcupine'
```
 or is it from `launcher/main.c` but then to it is showing errors both on windows and linux.

Reply:
See instructions in `CONTRIBUTING.md`. The correct command is `py -m porcupine`.

Reply:
So I just ran the file and tested it myself. And it is working 🥳 along with being present in the present working directory.